firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/internal/common/IdManagerTest.java

```diff
@@ -75,6 +77,23 @@ public void testCreateUUID() {
     assertEquals(installId, idManager.getCrashlyticsInstallId());
   }
 
+  public void testGetIdExceptionalCase() {
```

Comment:
Can we make a minor name change here: `testGetIdExceptionalCase_doesNotRotateInstallId()`

Reply:
done